rocky/tools/models.py

```diff
@@ -54,6 +62,7 @@ class Organization(models.Model):
             "A slug containing only lower-case unicode letters, numbers, hyphens or underscores "
             "that will be used in URLs and paths"
         ),
+        validators=[organization_code_validator],
```

Comment:
Validators aren't run by `save` and I am not sure these validators are run by django-rest-framework. I think we should also add a test case to check that these codes are also rejected by the API.

Reply:
Once the organization code is created it cannot be changed, not from the admin panel + also not from api. Saving is here not used. I can use clean method here instead is also a possibility to also include save().

Reply:
Moved it to `clean` method